workspaces/ui-v2/src/optic-components/hooks/diffs/LearnInitialBodiesMachine.ts

```diff
@@ -174,14 +190,31 @@ export function recomputeCommands(ctx: InitialBodiesContext): any[] {
   if (ctx.stagedEndpointName) {
     commands.push(
       AddContribution(
-        getEndpointId({ method: ctx.method, pathId: ctx.pathId }),
+        getEndpointId({ method: ctx.method, pathId }),
         'purpose',
         ctx.stagedEndpointName
       )
     );
   }
 
-  return commands;
+  return commands.map((i) => {
```

Comment:
this is the part that feels funcky. I have to rewrite the pathIds to get this to work with the global commands